fin360/fin360.py

```diff
@@ -44,3 +44,16 @@ def authenticate(self, email, password):
         self.access_token = response.json()['access_token']
 
         return AttrDict(response.json())
+
+    def get_transactions_with_details(self, accountid):
+
+        url = "https://www.fin360.in/bank-account/api/v1/transactionsWithDetails/" + accountid + ".json"
```

Comment:
as i mentioned earlier, urls must be constructed from a base_url which is constant. Also use string formatting 
  
  `"https://www.fin360.in/bank-account/api/v1/transactionsWithDetails/{}.json".format(account_id)`